### PR TITLE
Update get-all-vulnerabilities.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated API description with odata filter queries

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5933

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/get-all-vulnerabilities.md [hunk 1]
```diff
@@ -34,7 +34,16 @@ ms.custom: api
...
+## API description
+
+Retrieves a list of all vulnerabilities.
+
+<br>Supports [OData V4 queries](https://www.odata.org/documentation/).
+<br>OData supported operators:
+<br>```$filter``` on:  ```id```, ```name```, ```description```, ```cvssV3```, ```publishedOn```, ```severity``` and ```updatedOn``` properties.
+<br>```$top``` with max value of 10,000 
+<br>```$skip```
+<br>See examples at [OData queries with Microsoft Defender for Endpoint](exposed-apis-odata-samples.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-all-vulnerabilities.md]:
```suggestion
Retrieves a list of all vulnerabilities.
<br>Supports [OData V4 queries](https://www.odata.org/documentation/).
<br>OData supported operators:
<br>```$filter``` on:  ```id```, ```name```, ```description```, ```cvssV3```, ```publishedOn```, ```severity```, and ```updatedOn``` properties.
<br>```$top``` with max value of 10,000.
<br>```$skip```.
<br>See examples at [OData queries with Microsoft Defender for Endpoint](exposed-apis-odata-samples.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@siosulli Gentle reminder

**Pull request merged**